GitDepend/Commands/ListCommand.cs

```diff
@@ -60,10 +61,17 @@ public ReturnCode Execute()
 
         private void WriteDependency(Dependency dependency, string indent)
         {
-            _console.WriteLine($"{indent}- {dependency.Configuration.Name}");
-            foreach (var subDependency in dependency.Configuration.Dependencies)
+            if (string.IsNullOrEmpty(dependency.Configuration.Name))
```

Comment:
Is `dependency.Configuration` guaranteed to not be null when there is no GitDepend.josn file?